src/Mafin.Configuration/Providers/SettingsFile/SettingsFileConfigurationBuilderExtension.cs

```diff
@@ -0,0 +1,103 @@
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.FileProviders;
+
+namespace Mafin.Configuration.Providers.SettingsFile;
+
+/// <summary>
+/// Extension methods for <see cref="IConfigurationBuilder"/> to add <see cref="SettingsFileConfigurationSource"/>.
+/// </summary>
+public static class SettingsFileConfigurationBuilderExtension
+{
+    private static string DefaultSettingsFile =>
+        Path.Combine(Environment.CurrentDirectory, "Mafin.Configuration.json");
```

Comment:
What will happen if the "Mafin.Configuration.json" file is not created? Suggesting extracting the "Mafin.Configuration.json" value into a constant.

Reply:
> What will happen if the "Mafin.Configuration.json" file is not created?
```
---- System.IO.FileLoadException : Unable to load configuration from Mafin.Configuration.json
```